rialto_airflow/harvest/dimensions.py

```diff
@@ -31,37 +96,9 @@ def dois_from_orcid(orcid):
             yield doi_id
 
 
-def doi_orcids_pickle(authors_csv, pickle_file, limit=None) -> None:
-    """
-    Read the ORCIDs in the provided rialto-orgs authors.csv file and write a
-    dictionary mapping of DOI -> [ORCID] to a pickle file at the provided path.
-    """
-    df = pd.read_csv(authors_csv)
-    orcids = df[df["orcidid"].notna()]["orcidid"]
-    orcid_dois = {}
-
-    for orcid_url in orcids[:limit]:
-        orcid = normalize_orcid(orcid_url)
-        dois = list(dois_from_orcid(orcid))
-        orcid_dois.update({orcid: dois})
-
-    with open(pickle_file, "wb") as handle:
-        pickle.dump(invert_dict(orcid_dois), handle, protocol=pickle.HIGHEST_PROTOCOL)
-
-
-def publications_csv(dois, csv_file) -> None:
-    with open(csv_file, "w") as output:
-        writer = csv.DictWriter(output, publication_fields())
-        writer.writeheader()
-        for pub in publications_from_dois(dois):
-            logging.info(f"writing metadata for {pub.get('doi')}")
-            writer.writerow(pub)
-
-
 def publications_from_dois(dois: list, batch_size=200):
     """
-    Get the publications metadata for the provided list of DOIs and write as a
-    CSV file.
+    Get the publications metadata for the provided list of DOIs.
```

Comment:
the CSV creation had already been split out into a separate function, which i then ended up removing because it was no longer used after the dimensions harvesting was reimplemented.